### ISSUE TITLE
a11y > "Impact Receipt" should be <h1> on receipt page

### ISSUE THREAD

**Issue by ryanschaefer41** [vigetlabs/EcoCart#70]
It's currently `h3`. Should be an `h1` to meet the WCAG A standard (all pages must have an `h1`).

![image](https://user-images.githubusercontent.com/23317410/89206701-a728ed80-d587-11ea-826b-c6fb2bb339ee.png)
